lang/cs/Org.Apache.REEF.IMRU.Examples/OnREEFIMRURunTimeConfiguration.cs

```diff
@@ -72,9 +73,11 @@ internal static IConfiguration GetYarnIMRUConfiguration()
             IConfiguration imruClientConfig =
                 REEFIMRUClientConfiguration<TMapInput, TMapOutput, TResult>.ConfigurationModule.Build();
 
+            var fileSystemConfig = HadoopFileSystemConfiguration.ConfigurationModule.Build();
```

Comment:
The code is put in the case of run on Yarn. And it is just an example. Clients can follow it to provide their own configurations.